packages/utils/src/saml/bulk-saml.ts

```diff
@@ -1,17 +1,18 @@
-// This is a helper script that lets you send xcpd requests in bulk
+import * as dotenv from "dotenv";
+dotenv.config();
 import fs from "fs";
-
 import { createAndSignBulkXCPDRequests } from "@metriport/core/external/saml/xcpd/iti55-envelope";
 import { sendSignedRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/saml-client";
 import { getEnvVarOrFail } from "@metriport/core/util/env-var";
 
-import * as dotenv from "dotenv";
-dotenv.config();
+/*This is a helper script that lets you sign and send xcpd requests in bulk to as many gateways as you want.
+ *Its purpose is to allow you test specific gateways responses with granularity when debugging or testing.
+ */
```

Comment:
nit: proper formatting of TSDoc - https://tsdoc.org/pages/tags/example/

Reply:
file isnt useful anymore and was deleted so no longer relevant